16.0/Dockerfile

```
@@ -56,7 +56,7 @@ RUN python -m venv $VIRTUAL_ENV
 ENV PATH="$VIRTUAL_ENV/bin:$PATH"
 
 # Install debugger
-RUN pip install setuptools wheel debugpy pre-commit packaging
+RUN pip install setuptools wheel debugpy pre-commit websocket-client
```

Hadolint (2.14.0)

[warning] Pin versions in pip. Instead of `pip install <package>` use `pip install <package>==<version>` or `pip install --requirement <requirements file>`

(DL3013)